qiita_pet/templates/sitebase.html

```diff
@@ -33,15 +33,15 @@
     <script>
       function overlay_check() {
         // from http://stackoverflow.com/q/5916900/379593
-        var ua=navigator.userAgent,tem,M=ua.match(/(opera|chrome|safari|firefox|msie|trident(?=\/))\/?\s*(\d+)/i) || []; 
+        var ua=navigator.userAgent,tem,M=ua.match(/(opera|chrome|safari|firefox|msie|trident(?=\/))\/?\s*(\d+)/i) || [];
         if(/trident/i.test(M[1])){
-            tem=/\brv[ :]+(\d+)/g.exec(ua) || []; 
+            tem=/\brv[ :]+(\d+)/g.exec(ua) || [];
             return 'IE '+(tem[1]||'');
-            }   
+            }
```

Comment:
since this editing is happening, please align this with the if statement. same for the one below.